diff --git a/node_modules/expo-notifications/src/DevicePushTokenAutoRegistration.fx.ts b/node_modules/expo-notifications/src/DevicePushTokenAutoRegistration.fx.ts
index a352c21..8edeca8 100644
--- a/node_modules/expo-notifications/src/DevicePushTokenAutoRegistration.fx.ts
+++ b/node_modules/expo-notifications/src/DevicePushTokenAutoRegistration.fx.ts
@@ -112,7 +112,7 @@ if (ServerRegistrationModule.getRegistrationInfoAsync) {
   // Verify if persisted registration
   // has successfully uploaded last known
   // device push token. If not, retry.
-  ServerRegistrationModule.getRegistrationInfoAsync().then(__handlePersistedRegistrationInfoAsync);
+  ServerRegistrationModule.getRegistrationInfoAsync().then(__handlePersistedRegistrationInfoAsync).catch(() => {});
 } else {
   console.warn(
     `[expo-notifications] Error encountered while fetching auto-registration state, new tokens will not be automatically registered on server.`,
